include/seqan3/alphabet/adaptation/char.hpp

```diff
@@ -98,7 +100,7 @@ namespace seqan3
 // ------------------------------------------------------------------
 
 /*!\brief Specialisation of seqan3::underlying_char for char types.
- * \tparam char_type One of `char`, `char16_t` or `char32_t`.
+ * \tparam char_type One of `char`, `char16_t` , `char32_t` or `wchar_t`.
```

Comment:
Please remove the whitespace before the comma.

(Same for the other 8 changes)